cssdata.py

```diff
@@ -0,0 +1,51 @@
+# import from existing libraries
+import pandas as pd
+from matplotlib import pyplot as plt
+import json
+
+def csv_to_dataframe(input, exp_id, trial_id, num_headers=5):
+    """
+    convert csv to dataframe based on the specified exp_id and trial_id
+    note that `input.json` data must be provided to input argument
+    :param input: `input.json` data
+    :param exp_id: experiment id
+    :param trial_id: trial id
+    :param num_headers: number of headers in csv file
+    :return: dataframe
+    """
+    experiments = input['experiments']
+    for experiment in experiments:
+        # iterate over experiments and find the experiment that corresponds to `exp_id` specified.
+        if experiment['exp_id'] == exp_id:
+            trials = experiment['trials']
+        for trial in trials:
+            # iterate over trials and fine the trial that corresponds to `trial_id` specified.
+            if trial['id'] == trial_id:
+                file = trial['file']
+                relative_density = trial['relative_density']
+                # convert csv to dataframe
+                df = pd.read_csv(f"./rawdata/Experiment-{exp_id}/{file}", header=num_headers)

```

Comment:
Why are you still adding `./rawdata/Experiment-stuff!` shouldn't this be read from the input JSON?